content/influxdb/clustered/install/deploy.md

```diff
@@ -9,6 +9,46 @@ menu:
 weight: 104
 ---
 
+Use Kubernetes and related tools to deploy your InfluxDB cluster.
+This guide provides instructions for deploying your InfluxDB cluster using the
+following tools:
+
+- **kubectl**: CLI for controlling the Kubernetes cluster manager
+- **kubit**: A Kubernetes controller that can render and apply jsonnet
+  templates based on the [kubecfg](https://github.com/kubecfg/kubecfg) jsonnet
+  tooling and framework
+
+InfluxDB Clustered uses an `AppInstance` Kubernetes custom resource (CR) to
+configure and deploy your InfluxDB Cluster.
+Installing a CR requires cluster-wide permissions and may cause `kubectl` to
+fail if you do not have those permissions in your cluster.
+
+`kubit` uses your local `kubectl` credentials to install CRs.
+This still requires you to have permissions to install cluster-wide resources
+(CRs), but doesn’t require `kubit` to be installed with those permissions
+in the cluster. 
+
+{{% note %}}
+**If you meet any of the following criteria, [install and use the `kubit` CLI](https://github.com/kubecfg/kubit#cli-tool)
+on your local machine to use it as an operator from your local terminal and
+deploy your cluster:**
```

Comment:
```suggestion
**If you meet any of the following criteria, [install and use the `kubit` CLI](https://github.com/kubecfg/kubit#cli-tool)
on your local machine. This allows you to act as the operator would and deploy your cluster, but from your terminal**
```

My suggested wording might not be the best, but I think this makes more sense as the `kubit local apply` steps are there to allow a user to act as though _they_ were the operator instead from their terminal.